cmd/dm-ctl/main.go

```diff
@@ -119,8 +119,22 @@ func main() {
 		}
 		os.Exit(0)
 	default:
-		common.PrintLinesf("parse cmd flags err: %s", terror.Message(err))
-		os.Exit(2)
+		// NOTE:  when `--help` in cmdArgs we need to print out the help msg.)
+		var needPrintHelp bool
+		for _, cmd := range cmdArgs {
+			if cmd == "--help" {
+				needPrintHelp = true
```

Comment:
seems `ctl.PrintHelp(cmdArgs)` and `os.Exit(0)` could be moved here. Also we don't need `needPrintHelp`